server/master/src/main/java/org/apache/accumulo/master/tableOps/CreateInitialSplits.java

```diff
@@ -99,10 +95,10 @@ private void writeLastRowToMetadataTable(SplitEntry splitEntry, BatchWriter bw)
       throws MutationsRejectedException {
     Text lastRow = new Text(tableInfo.tableId.toString() + "<");
     Mutation mut = new Mutation(lastRow);
-    mut.put(new Text("srv"), new Text("dir"), splitEntry.dirValue);
-    mut.put(new Text("srv"), new Text("lock"), splitEntry.lockValue);
-    mut.put(new Text("srv"), new Text("time"), splitEntry.timeValue);
-    mut.put(new Text("~tab"), new Text("~pr"),
+    MetadataSchema.TabletsSection.ServerColumnFamily.DIRECTORY_COLUMN.put(mut, splitEntry.dirValue);
```

Comment:
Could do static imports of these to make code shorter.  In eclipse may be able to do this with Ctrl+Shift+M

Reply:
done